src/components/AppHeader/AppHeader.tsx

```diff
@@ -1,56 +1,76 @@
+export const AppHeader: FunctionComponent<any> = (props) => {
+  const userContext = useContext(UserContext);
+  let navigate = useNavigate();
 
-    const userContext = useContext(UserContext);
-    let navigate = useNavigate();
+  function onNewClick(
+    ev?: MouseEvent<HTMLElement> | KeyboardEvent<HTMLElement> | undefined,
+    item?: IContextualMenuItem | undefined
+  ) {
+    navigate("new");
+  }
 
-    function onNewClick (ev?: React.MouseEvent<HTMLElement, MouseEvent> | React.KeyboardEvent<HTMLElement> | undefined, item?:IContextualMenuItem | undefined) 
-    {
-      navigate("new");
-    }
+  function onHomeClick(
+    ev?: MouseEvent<HTMLElement> | KeyboardEvent<HTMLElement> | undefined,
+    item?: IContextualMenuItem | undefined
+  ) {
+    navigate("");
+  }
 
-    function onHomeClick (ev?: React.MouseEvent<HTMLElement, MouseEvent> | React.KeyboardEvent<HTMLElement> | undefined, item?:IContextualMenuItem | undefined) 
+  const _items: ICommandBarItemProps[] = [
     {
-      navigate("");
-    }
-
-    const _items: ICommandBarItemProps[] = [
-        {
-            key: "title",
-            text: process.env.REACT_APP_TEST_SYS === 'true' ? 'In-Out-Process TEST' : 'In-Out-Process',
-            onClick: onHomeClick,
-        },
-        {
-          key: 'newItem',
-          text: 'New',
-          iconPros: {iconName: 'Add'},
-          onClick: onNewClick,
-        }
-        ];
-
-    const _farItems: ICommandBarItemProps[] =[
-            {
-              onRenderMenuIcon: () => null,
-              onRenderChildren: () => <Avatar image={{src: userContext.user?.imageUrl}} name={userContext.user?.Title} size={32}  />,
-              key: 'persona',
-              name: userContext.user?.Title,
-              iconOnly: true
-            }
-        ]          
-
-
+      key: "title",
+      text:
+        process.env.REACT_APP_TEST_SYS === "true"
+          ? "In-Out-Process TEST"
+          : "In-Out-Process",
+      onClick: onHomeClick,
+    },
+    {
+      key: "newItem",
+      text: "New",
+      iconPros: { iconName: "Add" },
+      onClick: onNewClick,
+    },
+  ];
 
-    return (
-        <Sticky stickyPosition={StickyPositionType.Header}>
-        <div role="heading" aria-level={1}>
-            <CommandBar items={_items} farItems={_farItems} ></CommandBar>
-        </div>
-      </Sticky>        
+  const _farItems: ICommandBarItemProps[] = [
+    {
+      onRenderMenuIcon: () => null,
+      onRenderChildren: () => (
+        <Avatar
+          image={{ src: userContext.user?.imageUrl }}
+          name={userContext.user?.Title}
+          size={32}
+        />
+      ),
+      key: "persona",
+      name: userContext.user?.Title,
+      iconOnly: true,
+    },
+  ];
 
-    );
-}
+  return (
+    <Sticky stickyPosition={StickyPositionType.Header}>
+      <div role="heading" aria-level={1}>
+        <CommandBar items={_items} farItems={_farItems}></CommandBar>
```

Comment:
Not sure a CommandBar is right here -- seems overly complicated, since we're not sending any commands to the current page/subcomponents. Probably just need some styled react-router <NavLink /> components. Not something to handle now, but maybe worth creating an issue for and working in the future?

Reply:
Not sure it is worth leveraging a separate issue for.  The header can be re-imagined as we add onto it with different options depending on what they what/need, and as FluentUI releases more  components.  Their v9 components leverage more readable children style vs passing everything in as a prop.

Reply:
I'm saying it's a miss-use of the component. Like using a screwdriver as a hammer. Sure, you can bang on a nail with the handle of a screwdriver, **and it'll work!** But we'd be better off using a hammer. In this case we're not _sending commands_ within the context of the current page (like making text bold, italic, underlined, bulleted, etc.). We're just clicking navigation links to get to different routes _which will change the context of the page._ Let's use the right tool for job: NavLink components.

Reply:
I get it now, created issue #18